docs/examples/general/data_loading/numpy_reader.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>I think that warning that the benefit is available only at particular setting, but that setting is a default is a bit awkward. I'd rather say that setting this flag will negate that benefit:</p><p> </p><p>Numpy reader allows <strong>the user</strong> to specify a region of interest, equivalent to the arguments specified to <code>slice</code> operation. The benefit is that the reader will only read the relevant part of the file, saving I/O <strong>bandwith</strong> and memory utilization. Note that setting <code>mmap</code> argument to <code>False</code> will negate that performance benefit, with the entire file being read first and then sliced.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>